bundles/org.openhab.binding.melcloud/src/main/java/org/openhab/binding/melcloud/internal/MelCloudBindingConstants.java

```diff
@@ -42,21 +43,31 @@ public class MelCloudBindingConstants {
     // List of all Channel ids
     public static final String CHANNEL_POWER = "power";
     public static final String CHANNEL_OPERATION_MODE = "operationMode";
+    public static final String CHANNEL_OPERATION_MODE_STRING = "operationModeString";
     public static final String CHANNEL_SET_TEMPERATURE = "setTemperature";
     public static final String CHANNEL_FAN_SPEED = "fanSpeed";
     public static final String CHANNEL_VANE_HORIZONTAL = "vaneHorizontal";
     public static final String CHANNEL_VANE_VERTICAL = "vaneVertical";
     public static final String CHANNEL_SET_TEMPERATURE_ZONE1 = "setTemperatureZone1";
     public static final String CHANNEL_ROOM_TEMPERATURE_ZONE1 = "roomTemperatureZone1";
+    public static final String CHANNEL_SET_TEMPERATURE_ZONE2 = "setTemperatureZone2";
+    public static final String CHANNEL_ROOM_TEMPERATURE_ZONE2 = "roomTemperatureZone2";
     public static final String CHANNEL_FORCED_HOTWATERMODE = "forcedHotWaterMode";
     public static final String CHANNEL_TANKWATERTEMPERATURE = "tankWaterTemperature";
+    public static final String CHANNEL_TANK_TARGET_WATER_TEMPERATURE = "tankTargetWaterTemperature";
+
+    public static final String CHANNEL_HEAT_TEMPERATURE_MODE_ZONE1 = "heatTemperatureModeZone1";
+    public static final String CHANNEL_HEAT_FLOW_TEMPERATURE_ZONE1 = "heatFlowTemperatureZone1";
+    public static final String CHANNEL_HEAT_TEMPERATURE_MODE_ZONE2 = "heatTemperatureModeZone2";
+    public static final String CHANNEL_HEAT_FLOW_TEMPERATURE_ZONE2 = "heatFlowTemperatureZone2";
 
     // Read Only Channels
     public static final String CHANNEL_ROOM_TEMPERATURE = "roomTemperature";
     public static final String CHANNEL_LAST_COMMUNICATION = "lastCommunication";
     public static final String CHANNEL_NEXT_COMMUNICATION = "nextCommunication";
     public static final String CHANNEL_HAS_PENDING_COMMAND = "hasPendingCommand";
     public static final String CHANNEL_OFFLINE = "offline";
+    public static final String CHANNEL_RETURN_TEMPERATURE_ZONE1 = "returnTemperatureZone2";
```

Comment:
zoen 1 or zone  2 ?
Either way, i would have expected this channel to be here for both zones.